whitepapers/Tables.md

```diff
@@ -1,5 +1,114 @@
 # SQL Tables
 
+## Template
 
+### {ModelName}
+{Description}
 
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| | | ✓ | ✓ | ✓ | | |
 
+#### Other indices
+* `{column_name}`, `{column_name}`, ... [(unique)]
+* ...
+
+## Data
+
+### SourceUniqueIdentifier (SUID)
+Identifier for a specific document from a specific source.
+
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| `source_doc_id` | text |  |  |  |  | Identifier given to the document by the source |
+| `ingest_config_id` | int |  |  | ✓ |  | IngestConfig used to ingest the document |
```

Comment:
Stop me if I'm worrying too much about an absurdly unlikely case (or am just confused), but what if a source has two APIs (or their OAI endpoint supports two metadata prefixes, or whatever) which largely overlap, but each has information the other lacks:
* A has good contributor affiliations
* B has good funding information

If we harvest from both, the Consolidator will look at freshly transformed data from A and think "oh, that funding information must have been deleted", then look at data from B and think, "oh, these affiliations must have been removed", and we'll never have both at the same time.